fedimg/consumers.py

```diff
@@ -63,14 +63,19 @@ def _get_upload_urls(self, builds):
         # Get all of the .raw.xz URLs for the builds
         if len(builds) == 1:
             task_result = koji_session.getTaskResult(builds[0])
-            rawxz_files.append(get_rawxz_url(task_result))
+            url = get_rawxz_url(task_result)
+            if url:
+                rawxz_files.append(url)
         elif len(builds) >= 2:
             koji_session.multicall = True
             for build in builds:
                 koji_session.getTaskResult(build)
             results = koji_session.multiCall()
             for result in results:
-                rawxz_files.append(get_rawxz_url(result[0]))
+                url = get_rawxz_url(result[0])
```

Comment:
Yeah, but it's not the ``IndexError`` we've been seing in the logs.  Nonetheless, it would be good to check ``if result`` first.